### DISCUSSION TITLE
Spoolman

### DISCUSSION THREAD

**Discussion by markomenino** [Donkie/Spoolman#136] [Q&A]
Goodnight
I'm having trouble getting it to work.
Someone can help me?

**Comment by Disane87** [comment]
What's your exact problem? Documentation should be enough for setting it up.

**Comment by markomenino** [comment]
The docker-compose.yml I installed does not have spoolman.db

**Comment by Disane87** [comment]
Did you use the compose file here and changed the path according to your system?

> If you cannot find this file in your machine, then your data will be lost every time you update Spoolman.

**Comment by markomenino** [comment]
The file I take here comes in .yaml
I'm sorry but I don't understand much of this

**Comment by Disane87** [comment]
You should take the file content which @Donkie has described in the readme. You have to change paths were you want to store the files. See the comments in the example composer file. 

If you want to know more about this topic, I would recommend to get into docker basics, since this is not a real problem of spoolman.
